dc-commons-file/src/main/java/de/digitalcollections/commons/file/backend/impl/FileResourceRepositoryImpl.java

```diff
@@ -158,7 +173,9 @@ public FileResource find(String key, FileResourcePersistenceType resourcePersist
   public InputStream getInputStream(URI resourceUri) throws ResourceIOException {
     try {
       String location = resourceUri.toString();
-      LOGGER.info("Getting inputstream for location '{}'.", location);
+      if (LOGGER.isDebugEnabled()) {
```

Comment:
guard not really necessary as no costly operation in logging statement, see https://stackoverflow.com/a/12953090